drivers/wifi/siwx917/siwx917_wifi.c

```diff
@@ -195,21 +195,32 @@ static unsigned int siwx917_on_scan(sl_wifi_event_t event, sl_wifi_scan_result_t
 static int siwx917_scan(const struct device *dev, struct wifi_scan_params *z_scan_config,
 			scan_result_cb_t cb)
 {
-	sl_wifi_scan_configuration_t sl_scan_config = { };
+	sl_wifi_scan_configuration_t sl_scan_config = {0};
```

Comment:
This seems like an unnecessary change to me. `{}` should be perfectly fine to be used as an initializer.